intro.md

```diff
@@ -0,0 +1,17 @@
+Welcome to Win32 App Isolation Introduction Page
+
+Target Applications
+This feature is intended to increase the security of Win32 and Centennial apps.
+
+Minimum requirements:
+•	Check that Windows version is 25357 or greater
+•	One will need to download both MPT v1.2023.517.0 and ACP (Releases · microsoft/win32-app-isolation (github.com))
```

Comment:
The way to create a hyperlink in md file is

```
[the word appears](https://actuallink.com)
```

Also I think we should separate MPT and ACP. We should mention that ACP and WPR is needed if they need to know which capabilities are required for their apps. It's actually not "necessary" to package their apps.